anda/misc/prime-run/prime-run.spec

```
@@ -0,0 +1,38 @@
+Name:           prime-run
+Version:        0.0.1
```

```suggestion
Version:        1.0
```
Probably want to follow other distros' versioning